src/app/shared/store/registration.state.ts

```
@@ -192,33 +183,59 @@ export class RegistrationState {
+          this.router.navigate(['/forbidden']);
+        }
+        return throwError(() => new Error('Something went wrong!'));
+      }),
+      finalize(() => patchState({ isAuthorizationLoading: false }))
+    );
   }
 
   @Action(GetUserPersonalInfo)
-  getUserPersonalInfo({ patchState }: StateContext<RegistrationStateModel>): Observable<User> {
+  getUserPersonalInfo({ patchState }: StateContext<RegistrationStateModel>): Observable<User> | void {
     patchState({ isLoading: true });
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)